settings.gradle

```diff
@@ -1 +1,22 @@
+pluginManagement {
+    gradle.ext.agpVersion = '7.1.0'
```

Comment:
Not sure what would happen since the android app still uses `4.2.2` (we are working on moving to `7.1.1` soon. So if you want you can update this to `7.1.1`. We will see if the app version overrides this one, otherwise we might need to delay using this version a bit

Reply:
Since PermissMe is becoming a binary dependency, there is no strict requirement to keep the same version of AGP. The app won't override this one, because we won't be building PermissMe as part of the client, it'll already be built. However, if we start using [composite build](https://docs.gradle.org/current/userguide/composite_builds.html) for development (which is something used for other a8c clients) then it becomes important to keep them aligned.

There _might_ still be some transitive effects of using AGP that might effect things even as a binary dependency, but if this happens, it'd more likely to be one-off type issues. Otherwise, the whole ecosystem would have to use the same AGP version which is not possible.

Having said all that, I did upgrade AGP to `7.1.1` in 90d4b3810f0675daf1ba02f4991d7ccdd25cd342. I think this is a good change regardless and the only reason I previously opted for `7.1.0` was because I thought it was the latest release. (maybe it was back then?)

We also don't have to merge these PRs immediately and can wait for the AGP upgrade of the client if that's something you prefer.